source/develop/release-management/features/storage/hotplugdisk.html.md

```diff
@@ -17,65 +17,42 @@ feature_status: Released
 
 ### Summary
 
-Allow to hot add and hot remove of a virtio-blk disks from or to running guest.
-
-### Owner
-
-*   Michael Kublin
-*   Email: mkublin@redhat.com
```

Comment:
This email no longer exists, so I'm not sure what's the point of keeping it, but you're right - such decisions should be made in discussions, not patches. I'll revert this change.